test/e2e/e2e_test.go

```diff
@@ -73,7 +73,8 @@ var _ = Describe("controller", Ordered, func() {
 
 			By("installing CRDs")
 			cmd = exec.Command("make", "install")
-			_, err = utils.Run(cmd)
+			// TODO: Handle CRD installation error
+			_, _ = utils.Run(cmd)
```

Comment:
Is it still makes sense after https://github.com/aenix-io/etcd-operator/pull/15 get merged?

Reply:
Yep, it's a linter validation issue.